utils/initial_data_preparation.py

```diff
@@ -314,3 +336,18 @@ def get_base_repo_url(url):
                 f"Saved {domain} domain DataFrame with "
                 f"{len(domain_df)} entries to {data_folder}"
             )
+
+        # Append DataFrames with less than 10 repositories to the
+        # other_domains_df
+        else:
+            other_domains_df = pd.concat(
+                [other_domains_df, domain_df], ignore_index=True
+            )
+
+    # Save the DataFrame containing domains with less than 10 repositories
+    if not other_domains_df.empty:
+        other_domains_df.to_csv(data_folder / "other_domains.csv", index=False)
+        logger.info(
+            f"Saved DataFrame with domains having less than 10 repositories to: "
```

Comment:
Again, it's probably worth using a consistent constant value for this number (10) throughout our codebase.